src/polycubed/src/rest_server.cpp

```diff
@@ -931,5 +937,157 @@ const std::string &RestServer::getPort() {
     return port;
 }
 
+void RestServer::create_metrics() {
```

Comment:
Write a small comment about what this function is about.

Reply:
Added :+1:  Sorry but some comments were in an old version of my code, thanks for the osservation.